notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>Consider only including a few of these and linking to documentation for the rest.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/726/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
This feature is in private preview and documentation is a work in progress. When it's finished, I'll remove this and link to that directly.

Reply:
<p>I think I'm the one responsible for the documentation and I haven't started it (other than the small section under https://cloud.google.com/vertex-ai/docs/tabular-data/tabular-workflows/overview). Why don't you list just those parameters that you'll be setting in this notebook? I assume there are default values for most of these. </p>

Reply:
<p>I understand it's a bit unwieldy, but I think we should keep this here until we can point to documentation. The parameters listed here that we do not explicitly set/reference are related to machine type (stats_and_example_gen_dataflow*, transform_dataflow*, evaluation_dataflow*, and evaluation_batch_predict*), or about 12 of them. If you feel strongly, I can remove those, but I prefer to keep them.</p>

Reply:
<p>Sakshi, how do you think for now we remove the unused parameters. And in parallel update the docstring in GCPC (example CL: cl/462205490). So after the next GCPC release, the documentation will be at [1], we can update the notebook to point to there.</p><p> </p><p>[1] https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0.13/google_cloud_pipeline_components.experimental.automl.tabular.html</p><p> </p>